lib/screens/app/process_dynamic_links.dart

```diff
@@ -0,0 +1,135 @@
+import 'package:flutter/material.dart';
+import 'package:http/http.dart';
+import 'package:provider/provider.dart';
+import 'package:seeds/features/scanner/telos_signing_manager.dart';
+import 'package:seeds/providers/notifiers/settings_notifier.dart';
+import 'package:seeds/providers/services/esr_service.dart';
+import 'package:seeds/providers/services/links_service.dart';
+import 'package:seeds/screens/app/wallet/custom_transaction.dart';
+import 'package:url_launcher/url_launcher.dart' as UrlLauncher;
+import 'package:convert/convert.dart';
+
+class ProcessDynamicLinks extends StatefulWidget {
```

Comment:
This entire class is a view, 

There should be no network requests here
No Logic
No mappings 
Etc


This class should have a bloc that handles all these things. 

https://pub.dev/packages/flutter_bloc#:~:text=BlocProvider%20is%20a%20Flutter%20widget,multiple%20widgets%20within%20a%20subtree.


We have many examples how to integrate blocs into our views in V2. I am also here to help with any questions.